src/vm/perfinfo.h

```diff
@@ -0,0 +1,26 @@
+#ifndef PERFINFO_H
```

Comment:
Please add the file header at the top.  You can find it in any of the existing source files.